.github/workflows/stale.yml

```
@@ -0,0 +1,50 @@
+name: Check stale issues and pull requests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 12 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+env:
+  BEFORE_ISSUE_STALE: 180
+  BEFORE_ISSUE_CLOSE: 0 #FIXME: change to 14 days
+  BEFORE_PR_STALE: 14
+  BEFORE_PR_CLOSE: 7
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6
+        with:
+          debug-only: true
+          days-before-issue-stale: ${{ env.BEFORE_ISSUE_STALE }}
+          days-before-issue-close: ${{ env.BEFORE_ISSUE_CLOSE }}
+          stale-issue-message: |
+            Hi everyone! This issue has been marked as stale due to inactivity for the last ${{ env.BEFORE_ISSUE_STALE }} days.
+            It will be automatically closed in ${{ env.BEFORE_ISSUE_CLOSE }} days.
+            If you think this issue is still relevant in the latest Solidity version or you have something to [contribute](https://docs.soliditylang.org/en/latest/contributing.html) feel free to reopen it.
```

I think the last line should be in the close message instead. Here the issue is not closed and actually, only the submitter can reopen so addressing this to everyone is also a bit misleading.
```suggestion
          stale-issue-message: |
            Hi everyone! This issue has been marked as stale due to inactivity for the last ${{ env.BEFORE_ISSUE_STALE }} days.
            It will be automatically closed in ${{ env.BEFORE_ISSUE_CLOSE }} days.
```